site/src/pages/index.tsx

```diff
@@ -12,6 +12,7 @@ import { Button } from "@jpmorganchase/uitk-core";
 import styles from "./index.module.css";
 import Features from "../components/features/Features";
 import Card, { CardProps } from "../components/card/Card";
+import Accordion from "../components/accordion/Accordion";
```

Comment:
Given that our accordion "component" is very specific to the homepage - both in terms of its layout, illustration and the fact the text content is embedded in that component, it might make sense to move those files here under `pages/` so that our `components/` dir is only used for reusable things. That way all the files that make up the content of the homepage are also closer together.

[Docusaurus will ignore files and folders prefixed with an underscore](https://docusaurus.io/docs/creating-pages#routing), so perhaps we could create `site/src/pages/_index/` and move those files into there.

Reply:
Makes sense, I've moved both `Accordion` and `Features`